planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
Hi.

#257 and #259 are aligned. Both of them avoid NodeTypes, of course they are related between them. They allow to build and managed DAMs without NodeTypes definitions, and these DAMs can be deployed and it should work and they can work without policies so I think they are independent of #252. 

#252 adds policy and some Dashboard modifications, but we need to clarify the policy.  We need to clarify some policy points before adding to DAMs. #252 does not use `MonitoringDamGenerator` because the policy requires the generated MonitoringRules. When, we connected the `DamGenerator` and the Policy we will used `MonitoringDamGenerator`.

Of course, we could close #257 and #259 and move their changes to a new PR. Then, we will continue with policy.
WDYT?